tokio/src/lib.rs

```diff
@@ -339,14 +339,32 @@
 //! _parking_lot_ release in use.
 //!
 //! ### Unstable features
-//!
-//! These feature flags enable **unstable** features. The public API may break in 1.x
-//! releases. To enable these features, the `--cfg tokio_unstable` must be passed to
-//! `rustc` when compiling. This is easiest done using the `RUSTFLAGS` env variable:
-//! `RUSTFLAGS="--cfg tokio_unstable"`.
-//!
+//! 
+//! Some feature flags are only available when specifying the `tokio_unstable` flag:
+//! 
 //! - `tracing`: Enables tracing events.
+//! 
+//! Likewise, some parts of the API are only available with the same flag:
+//! 
+//! - [`task::JoinSet`]
```

Comment:
This is missing the task builder.

Reply:
Ah! I wasn't aware of that one. I'll add it. Thanks!